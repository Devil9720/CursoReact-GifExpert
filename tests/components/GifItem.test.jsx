import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem"


describe('evaluaciones para GifItem', () => { 

    const title = 'Goku';
    const url = 'https://goku.com';


    test('deberia tomar una snapshot de GifItem', () => { 
        render(<GifItem title={ title }  url={ url }/>);
        expect( render ).toMatchSnapshot();
        
     })

 })