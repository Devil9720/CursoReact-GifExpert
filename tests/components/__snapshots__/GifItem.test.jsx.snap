// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`evaluaciones para GifItem deberia tomar una snapshot de GifItem 1`] = `[Function]`;
